ConTabs/OutputBuilder.cs

```diff
@@ -50,6 +60,11 @@ private OutputBuilder(Table<T2> t, Style s)
                     InsertVerticalPadding(table.Padding.Bottom, style.Wall); NewLine();
                 }
                 HLine(TopMidBot.Bot);
+
+                foreach (var col in hiddenCols)
+                {
+                    col.Hide = false;
```

Comment:
I think this is the main reason I'd advocate using a separate property for this sort of "hiding". When we're done rendering the table, we want the columns we've hidden due to space constraints to be unhidden, but we don't want this for the ones where the _user_ did `.Hide = true`.

Reply:
It has been some time since I wrote it, but I think this is a naming issue. Variable hiddenCols does not represent all the columns that are hidden, just the columns that were hidden because of not being able to fit on the canvas. Because of that, it will not show the columns that were hidden manually by user. But just from this confusion, I agree that using Hide property was not a good idea and I should have used another property for that (probably internal, not a public one), that will be used just to suppress the columns if they can't fit.